### ISSUE TITLE
Missing image age metric for xubuntu-minimal

### ISSUE THREAD

**Issue by murraybd** [ubuntu/ubuntu-release-metrics#30]
We only have daily image age data for "xubuntu amd64" while there are actually two Xubuntu images being produced, https://cdimage.ubuntu.com/cdimage/xubuntu/, a daily-live and a daily-minimal. My basic testing of the images_collector.py script indicates that we are only we are only looking at path_table[0] and skipping over path_table[1] which contains 'daily-minimal' or 'daily-live'. Subsequently, the data may sometimes be the age and size of the daily-minimal or the daily-live.

When fixing this we'll need to think about the effects of modify an existing tag, "flavor", or adding a new one on the historical data.

**Comment by murraybd** [comment]
The same issue likely occurs with Ubuntu Budgie which has a daily-legacy and daily-live image.

**Comment by andersson1234** [comment]
yes, this seems to be the same issue as with the arm64 images. Though this requires more heuristics in the code to fix this issue specifically for xubuntu and budgie.

**Comment by Hyask** [comment]
As of now, `xubuntu-minimal` images correctly show up on the metrics, closing this. (Hi Brian! 🙂)
